docs/book/src/images/worker-machines-resources.plantuml

```diff
@@ -0,0 +1,46 @@
+@startuml worker-machines-resources
+
+skinparam object {
+    backgroundColor<< Infrastructure provider CRD >> #D9EAD3
+}
+
+' Cluster API CRD
+object Cluster << Cluster API CRD >>
+
+object MachineDeployment << Cluster API CRD >> {
+    spec.clusterName
+    <font color="Green">spec.template.spec.infrastructureRef</font>
+    spec.template.spec.bootstrap.configRef
+}
+
+object KubeadmConfigTemplate << Cluster API CRD >>
+
+object MachineSet << Cluster API CRD >> {
+    <font color="Green">spec.template.spec.infrastructureRef</font>
+    spec.template.spec.bootstrap.configRef
+}
+
+object Machine << Cluster API CRD >> {
+    <font color="Green">spec.infrastructureRef</font>
+    spec.bootstrap.configRef
+}
+
+object KubeadmConfig << Cluster API CRD >> 
+
+' Infrastructure provider CRD
+object FooMachineTemplate << Infrastructure provider CRD >>
+object FooMachine << Infrastructure provider CRD >>
+
+MachineDeployment "1..*" .right.> "1" Cluster
```

Comment:
The spacing between MachineDeployment and Cluster makes it difficult to discern the intended 1..* -> 1 relationship

Reply:
This should probably be updated to be 0..* -> 1, since MachineDeployments are not technically required, even if recommended.

Reply:
```suggestion
MachineDeployment "0..*" .right.> "1" Cluster
```

Reply:
I changed to take the distance between objects at 3986ef3.

![image](https://user-images.githubusercontent.com/230185/86257655-d1883380-bbf4-11ea-8ac4-bd29fc270423.png)